src/lib/wallets/xBull/XBull.ts

```diff
@@ -52,18 +55,10 @@ export default class XBull extends AbstractWallet implements IWallet {
 
     public override isInstalled(): Promise<boolean> {
         const xBullPromise: Promise<boolean> = new Promise((resolve) => {
-            window.addEventListener('message', (event) => {
-                if (event.data.type === XBull.XBullInjected && !!window.xBullSDK) {
-                    resolve(true);
-                }
-            });
-            setTimeout(() => {
-                if (window.xBullSDK) {
-                    resolve(true);
-                } else {
-                    resolve(false);
-                }
-            }, 100);
+            const isXBullInstalled = true;
+            if (isXBullInstalled) {
+                resolve(true);
+            }
```

Comment:
What's the purpose of this? Why not resolve(true); and be done with it?

Reply:
I did it for the sake of code homogeneity, as it's done this way on the PrivateKey class. I'm gonna change both.